WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -1142,6 +1149,15 @@ private extension AztecPostViewController {
             }
         }
 
+        if gutenbergSettings.isGutenbergEnabled(),
```

Comment:
It actually started like that, but I ended up putting it as a property to avoid instantiating it each time the button is pressed.

But I'll make the change,  since I don't really think instantiating it each time will really have any visible impact - so the optimization isn't really necessary.